Makefile

```diff
@@ -0,0 +1,30 @@
+SOURCE_FILES?=$$(glide novendor)
+TEST_PATTERN?=.
+TEST_OPTIONS?=
+
+setup: ## Install all the build and lint dependencies
+	curl https://glide.sh/get | sh
+	go get -u github.com/kisielk/errcheck
+	go get -u github.com/golang/lint/golint
+	glide install
+
+test: ## Run all the tests
+	go test $(TEST_OPTIONS) -cover $(SOURCE_FILES) -run $(TEST_PATTERN) -timeout=30s
+
+lint: ## Run all the linters
+	errcheck $(SOURCE_FILES)
+	go vet $(SOURCE_FILES)
+	golint ./... | grep -v vendor
+
+ci: lint test ## Run all the tests and code checks
+
+build: ## Build a beta version of releaser
+	go build
```

Comment:
I really meant only as an example. Right now, I have a project where I need to ember some assets into the binary and `go build` does not fit anymore so I ended up getting a `make build` anyway. Of course, if you do not see the benefit, I will take it out!